index.js

```diff
@@ -34,36 +36,50 @@
 // // => 11
 // console.log("%c----------", "color: red") 
 
-
+const drink_water = (currentThirstLevel) => { 
+  console.log("Man I sure am thirsty");
+  currentThirstLevel -= 1;
+  console.log("Ahh that hits the spot");
+  return currentThirstLevel;
+}
 
 // ***** Question 4 *****
 
 // *** Uncomment the lines below to test
 // console.log("%cQuestion 4", "color: red") 
 
-// console.log(sameSameButDifferent(5, 5)) 
+ 
+const sameSameButDifferent = (num , maybeNum) => {
+  if(num === maybeNum)
+    return 'same same (but different)';
+  else if(num == maybeNum) 
+    return 'same same';
+  return 'different';
+}
```

Comment:
Good use of strict (`===`) and loose (`==`) equality checks here! In terms of syntax, it's convention to use curly brackets to make a block for each if/else condition like this:

```js
  if (num === maybeNum) {
    return "same same"
  } else if (num == maybeNum) {
    return "same same (but different)"
  } else {
    return "different"
  }
```